src/common/modules/validator.cr

```diff
@@ -12,10 +12,7 @@
 
 module ::Axentro::Common::Validator
   def valid_amount?(amount : Int64) : Bool
-    if Int64::MAX < amount || 0 > amount
-      raise AxentroException.new("the amount is out of range")
-    end
-
+    raise InvalidAmount.new if amount < 0
```

Comment:
I'm not sure why there was this check : Int64::MAX < amount
I try something like Int64::MAX + 1_i64 and crystal automaticaly raise an exception so I think it's not necessary